documentation/asciidoc/topics/yaml/disable_fips_cr.yaml

```
@@ -0,0 +1,5 @@
+spec:
+  container:
+    extraJvmOpts: “-Dcom.redhat.fips=false"
+    cliExtraJvmOpts:  “-Dcom.redhat.fips=false"
+		routerExtraJvmOpts:  “-Dcom.redhat.fips=false"    
```

```suggestion
    extraJvmOpts: "-Dcom.redhat.fips=false"
    cliExtraJvmOpts: "-Dcom.redhat.fips=false"
    routerExtraJvmOpts: "-Dcom.redhat.fips=false"    
```
Adjust indent and spacing
All three fields should have the same indentation
The quotation marks looks strange to me
Remove the extra spaces